src/Pos/CLI.hs

```diff
@@ -283,17 +265,40 @@ walletPortOption portNum help =
         <> Opt.value portNum
         <> Opt.showDefault
 
-ipPortOption :: NetworkAddress -> Opt.Parser NetworkAddress
-ipPortOption na =
+externalNetworkAddressOption :: Maybe NetworkAddress -> Opt.Parser NetworkAddress
+externalNetworkAddressOption na =
```

Comment:
Here something obscure is going on. 
* I don't think it's needed to specify `port` here as the only thing we're having problems with is host redirection
* I've already implemented this functionality as part of CSL-870 PR and this particular commit: https://github.com/input-output-hk/cardano-sl/pull/185/commits/e2a25dfdf372aa05bbb6599843891819f7007261

I'll be able to merge CSL-870 with it later though i guess...